selfdrive/car/gm/interface.py

```diff
@@ -247,6 +247,16 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       ret.steerActuatorDelay = 0.2
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
+    elif candidate == CAR.ESCALADE_ESV_2019:
+      ret.minEnableSpeed = -1.  # engage speed is decided by pcm
+      ret.mass = 2739.
+      ret.wheelbase = 3.302
+      ret.steerRatio = 17.3
+      ret.centerToFront = ret.wheelbase * 0.5
+      ret.steerActuatorDelay = 0.2
+      CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
+      ret.tireStiffnessFactor = 1.0
```

Comment:
Can we merge with `ESCALADE_ESV`? No need for a new platform if we can help it.

Reply:
My only concern there is that we ended up with lat accel factor 1.3 based on what felt best, vs 1.9 for the other Escalade. That 50% difference will matter.